Makefile

```diff
@@ -112,13 +112,13 @@ set-max-map-count:
 .PHONY: deploy-es-operator
 deploy-es-operator: set-max-map-count
 	@kubectl create namespace ${ES_OPERATOR_NAMESPACE} 2>&1 | grep -v "already exists" || true
+	@kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheusrule.crd.yaml
```

Comment:
These have to be deployed before ES operator otherwise there are some error logs: 

> time="2019-03-21T11:05:55Z" level=error msg="error syncing key (selfprovisionedes-smoke-test-1553166300/elasticsearch): Failed to reconcile Service Monitors for Elasticsearch cluster: Failure constructing Elasticsearch ServiceMonitor: failed to get resource client: failed to get resource type: failed to get the resource REST mapping for GroupVersionKind(monitoring.coreos.com/v1, Kind=ServiceMonitor): no matches for kind \"ServiceMonitor\" in version \"monitoring.coreos.com/v1\""